src/app/app.tsx

```diff
@@ -45,6 +47,21 @@ export function App() {
   const [sunburstChartData, setSunburstChartData] =
     useState<SunburstData | null>();
 
+  const [selectedVersion, setSelectedVersion] = useUrlParam<string>(
+    'selectedVersion',
+    {
+      defaultValue: 'versions',
+      serializer: {
+        serialize: (v) => v,
+        deserialize: (s) => s,
+      },
+    }
+  );
```

Comment:
You shouldn't need to provide serializer here - this should look something like:
```typescript
  const [selectedVersion, setSelectedVersion] = useUrlParam<string>(
    'selectedVersion',
    'versions'
  );
```

Reply:
👍